### DISCUSSION TITLE
Translate Full Access

### DISCUSSION THREAD

**Discussion by TannyaGupta** [aws/serverless-application-model#3142] [Ideas]
Need to add translate full access policy as well

**Comment by aaythapa** [comment]
Hi @TannyaGupta can you provide more info (maybe a link to a doc for the policy)

**Comment by awood45** [comment]
Moving to discussions for feature requests - we will take a PR for this if interested in making one in the meantime.

**Comment by TannyaGupta** [comment]
Testing the step function that translate the code into target languages by invoke the step function but lambda need translate permission to access it locally.
Translate documentation: https://docs.aws.amazon.com/translate/latest/APIReference/API_StartTextTranslationJob.html.
IAM based policy documentation: https://docs.aws.amazon.com/translate/latest/dg/security_iam_id-based-policy-examples.html.
